docs/models/index.md

```diff
@@ -93,11 +108,11 @@ $ wget -P ~/autoware_data/traffic_light_fine_detector/ \
        https://awf.ml.dev.web.auto/perception/models/tlr_yolox_s/v2/tlr_yolox_s_batch_6.onnx \
        https://awf.ml.dev.web.auto/perception/models/tlr_yolox_s/v2/tlr_labels.txt
 
+# tvm_utility
 
-# traffic_light_ssd_fine_detector
+$ mkdir -p ~/autoware_data/tvm_utility/models/
+$ wget -p ~/autoware_data/tvm_utility/ \
+       https://autoware-modelzoo.s3.us-east-2.amazonaws.com/models/3.0.0-20221221/yolo_v2_tiny-x86_64-llvm-3.0.0-20221221.tar.gz
+$ tar -xvzf ~/autoware_data/tvm_utility/yolo_v2_tiny-x86_64-llvm-3.0.0-20221221.tar.gz -C ~/autoware_data/tvm_utility/models/
```

Comment:
Why does this tarball include a binary library? We should only download ONNX files here

Reply:
@esteve thanks for your comment. I update onnx and files for yolox and traffic_light and found that there is missing download instruction for tvm_utility. I followed [here ](https://github.com/autowarefoundation/autoware/blob/1f4d5d67462ed7a7ffd1239861141d00da700299/ansible/roles/artifacts/tasks/main.yaml#L381-L387) to update download instruction. 

I think those lines were added by https://github.com/autowarefoundation/autoware/pull/3916 @lexavtanke Did you have any intention to add library file to `autoware_data`